drv/lpc55-rng/src/main.rs

```diff
@@ -197,14 +224,95 @@ impl NotificationHandler for Lpc55RngServer {
     }
 }
 
+/// Load a type implementing HandoffData (and others) from a config::DataRegion.
+/// Errors will be reported in the ringbuf and will return in None.
```

Comment:
**nitpick:** Confused me for a second, I presume you meant to write "return None."